code/modules/mob/living/carbon/xenomorph/castes/defiler/abilities_defiler.dm

```diff
@@ -191,16 +214,32 @@
 
 /datum/action/xeno_action/select_reagent/action_activate()
 	var/mob/living/carbon/xenomorph/X = owner
-	var/list/available_reagents = X.xeno_caste.available_reagents_define
-	if(list_position < length(available_reagents))
-		list_position++
+	var/i = GLOB.toxin_type_list.Find(X.selected_reagent)
+	if(length(GLOB.toxin_type_list) == i)
+		X.selected_reagent = GLOB.toxin_type_list[1]
 	else
-		list_position = 1
-	X.selected_reagent = available_reagents[list_position]
+		X.selected_reagent = GLOB.toxin_type_list[i+1]
+
 	var/atom/A = X.selected_reagent
-	to_chat(X, "<span class='notice'>We will now slash with <b>[initial(A.name)]</b>.</span>")
+	to_chat(X, "<span class='notice'>We will now use <b>[initial(A.name)]</b>.</span>")
+	update_button_icon()
+	return succeed_activate()
+
+/*
+/datum/action/xeno_action/select_reagent/alternate_keybind_action()
+	var/toxin_choice = show_radial_menu(owner, owner, GLOB.toxin_images_list, radius = 48)
+	if(!toxin_choice)
+		return
+	var/mob/living/carbon/xenomorph/X = owner
+	for(var/toxin in GLOB.toxin_type_list)
+		var/datum/reagent/R = GLOB.chemical_reagents_list[toxin]
+		if(R.name == toxin_choice)
+			X.selected_reagent = R.type
+			break
+	to_chat(X, "<span class='notice'>We will now use <b>[toxin_choice]</b>.</span>")
 	update_button_icon()
 	return succeed_activate()
+*/
```

Comment:
?

Reply:
Linters doesn't like radials as alternate keybinds, and cycling is preferable as the primary.

Reply:
Linter doenst what? Add it back and just fix the linting issue.

Reply:
You can't fix the linting issue.

Reply:
if you can't fix the linting issue you're doing something wrong, the linter is there to block stuff done wrong.